functions/tips/requirements.txt

```diff
@@ -1,2 +1,6 @@
 google-cloud-error-reporting==0.30.0
-python-dateutil==2.7.3
\ No newline at end of file
+google-cloud-pubsub==0.35.4
+mock==2.0.0
```

Comment:
This is not needed as it's included in testing requirements.

Reply:
Done.